builds/misc/images.yaml

```diff
@@ -156,7 +156,7 @@ jobs:
       - template: templates/image-linux.yaml
         parameters:
           name: Metrics Collector
-          imageName: azureiotedge-metrics-collector
+          imageName: azureiotedge-test-metrics-collectorr
```

Comment:
Missed in my review, but I think we need to update this image name in all our deployment files.

Reply:
I think that I updated in all our deployment files.

Reply:
Oh yeah I missed it